XCUITest/FindInPageTest.swift

```diff
@@ -46,29 +46,4 @@ class FindInPageTest: BaseTestCase {
         // Ensure find in page bar is dismissed
         waitforNoExistence(element: app.buttons["FindInPage.close"])
     }
-    
-    func testActivityMenuFindInPageAction(){
```

Comment:
Did you mean to remove this? Find in page shouldn't have been affected by the redesign, right?

Reply:
Adding test back and updated comment. Should be updated in a separate PR.